src/vmm/src/device_manager/mmio.rs

```diff
@@ -86,6 +96,8 @@ pub struct MMIODeviceInfo {
 /// Manages the complexities of registering a MMIO device.
 pub struct MMIODeviceManager {
     pub(crate) bus: devices::Bus,
+    #[cfg(target_arch = "aarch64")]
+    pub gpio_pl061: Option<Arc<Mutex<devices::legacy::GPIO>>>,
```

Comment:
Would it make sense to create a `gpio` bus (which only currently has this device)?